packages/components/package.json

```diff
@@ -485,7 +485,7 @@
     "filesize": "^6.1.0",
     "framer-motion": "^11.0.3",
     "lodash": "^4.17.21",
-    "react-aria-components": "^1.7.1",
+    "react-aria-components": "^1.11.0",
```

Comment:
needed to support client side routing

<img width="1050" height="409" alt="image" src="https://github.com/user-attachments/assets/5d8d6f94-3fd6-4f96-a8cf-78ade1d57738" />

https://react-spectrum.adobe.com/react-aria/Menu.html#client-side-routing
https://react-spectrum.adobe.com/react-aria/routing.html#tanstack-router